README.md

```diff
@@ -283,13 +283,13 @@ Minima 2.x    | Minima 3.0
 
 You can add links to the accounts you have on other sites, with respective icon as an SVG graphic, via the config file.
 From `Minima-3.0` onwards, the social media data is sourced from config key `minima.social_links`. It is a list of key-value pairs, each entry
-corresponding to a link rendered in the footer. For example, to render links to Jekyll GitHub repository and twitter account, one should have:
+corresponding to a link rendered in the footer. For example, to render links to Jekyll GitHub repository and X account, one should have:
 
 ```yaml
 minima:
   social_links:
     - { platform: github,  user_url: "https://github.com/jekyll/jekyll" }
-    - { platform: twitter, user_url: "https://twitter.com/jekyllrb" }
+    - { platform: x, user_url: "https://x.com/jekyllrb" }
```

Comment:
Let's not change this. I don't think it's super important that the example match the new branding when the old branding still works.
```suggestion
    - { platform: twitter, user_url: "https://twitter.com/jekyllrb" }
```